client/rpc/src/system/tests.rs

```diff
@@ -333,3 +333,10 @@ fn system_network_remove_reserved() {
 	assert_eq!(runtime.block_on(good_fut), Ok(()));
 	assert!(runtime.block_on(bad_fut).is_err());
 }
+
+#[test]
+fn system_set_log_filter() {
+	sc_cli::init_logger("afg=debug", Default::default(), Default::default(), false).unwrap();
+	let res = api(None).system_set_log_filter("bla no ok -\\ /lba".into());
```

Comment:
What will this test?

Reply:
This, I believe, is a WIP.

Reply:
…but to expand on that, it's a "test" that essentially asserts that the expected behaviour is for the RPC calls to always succeed, i.e. return `HTTP 200`